web/app.js

```diff
@@ -864,11 +864,27 @@ const PDFViewerApplication = {
       .catch(downloadByUrl); // Error occurred, try downloading with the URL.
   },
 
+  _recordFallbackErrorTelemetry(featureId) {
+    if (
+      typeof PDFJSDev === "undefined" ||
+      PDFJSDev.test("MOZCENTRAL || GENERIC")
```

Comment:
Why `GENERIC` here as well? As far as I can see `reportTelemetry` is only implemented and useful in `MOZCENTRAL` builds.